src/doc/reference.md

```diff
@@ -3603,6 +3603,41 @@ fn main() {
 In this example, the trait `Printable` occurs as a trait object in both the
 type signature of `print`, and the cast expression in `main`.
 
+Trait objects may contain references, and so those references will need a
+lifetime. If the trait contains some sort of lifetime bound, like this:
```

Comment:
feels weird to me that this sentence just... ends (and doesn't resume after the code snippet)